.github/actions/use-android-emulator/action.yml

```diff
@@ -68,6 +71,7 @@ runs:
         avd-name: ${{ inputs.avd-name }}
         arch: x86_64
         target: google_apis
+        profile: pixel_7_pro # want a bigger screen to avoid UI element not found issue
```

Comment:
this is okay but i'm wondering if there were other approaches to make the snapshot more reliable. e.g.

having a fixed size container
```
<E2EContainer style={{ width: 600, height: 800 }} testId="e2eContainer">
  <TestView />
</E2EContainer>
```

and also scroll to the testId=e2eContainer.

is that feasible?